ct-app/.configs/core_local_config.yaml

```
@@ -116,8 +114,22 @@ sessions:
 # =============================================================================
 # 
 # =============================================================================
-fundings:
-  constant: "0 wxHOPR"
+investors:
+  schedule: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)